source/Calamari.Shared/Integration/Packages/Download/PackageDownloaderStrategy.cs

```diff
@@ -47,7 +49,8 @@ public PackageDownloaderStrategy(IScriptEngine engine, ICalamariFileSystem fileS
                     downloader = new GitHubPackageDownloader();
                     break;
                 case FeedType.Helm :
-                    downloader = new HelmChartPackageDownloader(fileSystem);
+                    var cred = feedCredentials.GetCredential(feedUri, "basic");
+                    downloader = new HelmChartPackageDownloader(fileSystem, new HelmEndpointProxy(new HttpClient(), feedUri, cred.UserName, cred.Password), new HttpClient());
```

Comment:
Since `feedCredentials` is a method argument to the downloader, can we get the downloader to pass it into the proxy via the method as well?

Reply:
Same with the Uri